packages/lodestar/src/sync/index.ts

```diff
@@ -103,7 +104,7 @@ export class Sync extends EventEmitter {
       opPool: this.opPool,
       logger: this.logger,
     });
-    this.syncer.start();
+    await this.syncer.start();
```

Comment:
I'm pretty sure this isn't suppose to be awaited, node should not wait for chain to synchronize